spectator-ext-ipc/src/main/java/com/netflix/spectator/ipc/IpcMetric.java

```diff
@@ -49,7 +49,9 @@ public enum IpcMetric {
           IpcTagKey.status
       ),
       EnumSet.of(
+          IpcTagKey.attemptReason,
```

Comment:
Another idea here that we should get more feedback on is if we should :
1. create a new set of `IpcMetricV2` enums so we separate the existing tag sets from the new tag sets
2. version this library and maintain the v1 separate from v2 (probably not enough here to warrant that)
3. keep this as-is understanding that the optional tags from v1 and v2 are slightly different